### PR TITLE
Update aggregator_placement.md

### PR DESCRIPTION
**Pull request by ejeannot**
Update for 2016-2017 report

### CODE REVIEW

File: _projects/aggregator_placement.md [hunk 1]
```diff
@@ -81,7 +87,8 @@ Remember to use the `--file jlesc.bib` with the `cite` tag.
 
 The next step will be to determine the parameters to consider to compute a near-optimal number of aggregators.
 Additionally, we plan to implement the topology-aware aggregator placement once a stable version of data aggregation will be developed.
-A new visit could be organized to that purpose. 
+We want to exted our approach to any kind of HPC system and especially to the new Theta system.
+Visists between Inria and ANL are planned for the summer 2016.
```
Comment (pancetta) [inline _projects/aggregator_placement.md:91]: This was during the reporting period. Did these visits happen? Or do you mean 2017?

**Review by pancetta — CHANGES_REQUESTED** [review]

**Comment by ejeannot** [comment]
Sorry I mean 2017. Can you change this!

  --Emmanuel

**Comment by pancetta** [comment]
Sure!

**Comment by ejeannot** [comment]
Thanks for everything. 

	—Emmanuel

**Pull request merged**